communications/telegram/telegram.go

```diff
@@ -25,17 +27,15 @@ const (
 	methodGetUpdates  = "getUpdates"
 	methodSendMessage = "sendMessage"
 
-	cmdStart    = "/start"
-	cmdStatus   = "/status"
-	cmdHelp     = "/help"
-	cmdSettings = "/settings"
+	cmdStart  = "/start"
```

Comment:
Suggestion: Would change time from local to UTC instead as server could be in diff zones.